### ISSUE TITLE
`ia list` should have a way to list filenames without CSV-style escaping

### ISSUE THREAD

**Issue by ethus3h** [jjjake/internetarchive#268]
The output format of `ia list` (when only printing a list of names, the default behavior) is hard to parse: it uses CSV-style escaping, for example:

```
"Constructed languages and universes/Ilien/Ilien/WikiIndex/Revision history of ""Ilien Wiki"" - WikiIndex.html.maff"
```
for a file named 
```
Constructed languages and universes/Ilien/Ilien/WikiIndex/Revision history of "Ilien Wiki" - WikiIndex.html.maff
```

This makes it hard to use in a shell script, because it requires a full CSV parser simply to iterate over the filenames.

It would be helpful to have a `-0` flag to output null-delimited sequences of bytes, with no escaping, as this is much easier to parse. (This is provided by various other command-line tools, such as `find`.)

**Comment by jakewilliami** [comment]
I know this is old, but would running `ia metadata` and parsing it through `jq` work?  E.g.,
```
ia metadata <identifier> | jq --raw-output '.files | .[] | .name'
```
